MonologBundle.php

```diff
@@ -50,4 +51,20 @@ public static function includeStacktraces(HandlerInterface $handler)
             $formatter->includeStacktraces();
         }
     }
+
+    /**
+     * @internal
+     */
+    public static function levelToMonologConst($level)
+    {
+        if (null === $level || is_numeric($level)) {
+            return $level;
+        }
+
+        if (defined('Monolog\Logger::'.strtoupper($level))) {
+            return constant('Monolog\Logger::' . strtoupper($level));
+        }
+
+        throw new \InvalidArgumentException(sprintf('Could not match "%s" to a log level.', $level));
```

Comment:
Thinking about it, the exception message could be a little more helpful if the raw string was injected (i.e. `%env(LOG_LEVEL)%`).